alert/alert.css

```diff
@@ -0,0 +1,50 @@
+
+.alert-container {
+    margin: 1rem;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    gap: 20px;
+}
+
+.alert-success,.alert-danger,.alert-info,.alert-warning{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
you can make separate class for this, everything else looks good !

Reply:
Sure I will note down it and refactor it at final stage